frontend/apps/filemanager/filemanager.lua

```diff
@@ -695,9 +720,10 @@ function FileManager:pasteHere(file)
                 self:moveFile(DocSettings:getSidecarDir(orig), dest) -- dest is always a directory
             end
             if self:moveFile(orig, dest) then
-                --update history
+                --update history and collections
```

Comment:
If we're updating comments, they like to be proper sentences. ;-)
```suggestion
                -- Update history and collections.
```